dockerfiles/ilias-configured/wait.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+should_run=true
+
+after_boot() {
+    if [ ! -z "$AFTER_BOOT" ]; then
+        chmod +x $AFTER_BOOT
+        "$AFTER_BOOT"
+        echo "After boot exited with code $?"
+    fi
+}
+
+before_shutdown() {
+    should_run=false
+    if [ ! -z "$BEFORE_SHUTDOWN" ]; then
+        chmod +x $BEFORE_SHUTDOWN
+        "$BEFORE_SHUTDOWN"
+        echo "Before shutdown exited with code $?"
+        exit $?
+    fi
+}
+
+wait_for() {
+    if [ ! -z "$WAIT_FOR" ]; then
+        for i in $(echo $WAIT_FOR | tr "," "\n")
+        do
+          array=$(echo $i | tr ":" "\n")
+          host="${array[0]}"
+          port="${array[1]}"
+          code=1
+          while [ "$code" != "0" ] && [ should_run ]
```

Comment:
Maybe you can shorten this to 
```bash
while ! $(nc -z -v -w5 $host $port)
do
    echo "Waiting for $i to come up..."
    sleep 5
done
```

For another project it worked for me like this: https://github.com/kekru/docker-wait-tcp/blob/master/wait

Reply:
Yes, we could also use your wait script to achieve the desired effect of depending containers. I don't mind :)

Reply:
I don't know which one works better, it just worked the last time :)